docs/en/query_language/functions/bitmap_functions.md

```diff
@@ -395,6 +395,77 @@ SELECT bitmapToArray(bitmapAndnot(bitmapBuild([1,2,3]),bitmapBuild([3,4,5]))) AS
 └───────┘
 ```
 
+## bitmapMin {#bitmapmin}
+
+Returns the smallest value in a bitmap.
+
+**Syntax**
+
+```sql
+bitmapMin(bitmap)
+```
+
+**Parameters**
+
+- `bitmap` — [Bitmap object](#bitmap_functions-bitmapbuild).
+
+**Returned values**
+
+- The smallest value in `bitmap`, if the set is not empty. Type: `UInt64`.
+- Maximum UInt32 number (4294967295), if the set is empty. Type: `UInt32`. 
```

Comment:
Result type can't be determined by runtime input values, so it's still UInt64. But returning 4294967295 is kind of weird decision, probably a bug.
```
SELECT
    bitmapMin(bitmapBuild(arrayFilter(x -> (x > 0), [0]) AS x)) AS y,
    x,
    toTypeName(y)
FORMAT TSV

4294967295	[]	UInt64
```